nx-os/poap/poap.py

```diff
@@ -102,7 +100,7 @@
 # script have many cases to test, going through a reboto takes too long)
 
 import sys
-import re
```

Comment:
One other thing, can we move these to the top of the file please, since you are updating things?